district-tasks/server/smart_contracts.cljs

```diff
@@ -0,0 +1,9 @@
+(ns district0x-tasks.shared.smart-contracts)
+
```

Comment:
are these addresses part of the repo you build in the containers? If not add the file before building